Signal/src/view controllers/SignalsViewController.m

```diff
@@ -339,25 +339,32 @@ - (void)tableViewCellTappedDelete:(NSIndexPath *)indexPath {
                       preferredStyle:UIAlertControllerStyleAlert];
         [self presentViewController:removingFromGroup animated:YES completion:nil];
```

Comment:
I think we can simplify the code and the UI a bit.

I don't think we wan't the user to see the alert controller unless we're leaving the group. Otherwise we can just delete the thread.